src/main/scala/sbtdatabricks/DatabricksHttp.scala

```diff
@@ -82,8 +82,77 @@ class DatabricksHttp(
     if (entity == null) null else EntityUtils.toString(entity)
   }
 
+  /*
+   * Check cluster
+   * @param clusterId of the desired cluster
+   * @return cluster 
+   */
+
+  private[sbtdatabricks] def checkCluster(clusterId: String): Option[Cluster] = {
+    val msg = s"Obtaining status information for Cluster Id $clusterId"
+    outputStream.println(msg)
+    val cluster = fetchClusters.filter(_.id == clusterId)
+    cluster.isEmpty match {
+      case false =>
+        val result = cluster.head
+        outputStream.println(result.toString)
+        Some(result)
+      case _ =>
+        outputStream.println(s"Cluster with Id '$clusterId' does not exist")
+        None 
+    }
+  }
+
   /**
-   * Upload a jar to Databrics Cloud.
+   * Create a cluster
+   * @param name of the cluster to be created
+   * @param memoryGB size in memory (GB) of the new cluster
+   * @param useSpot if true spot instances used, otherwise on-demand instances
+   * @return The id of the newly created cluster
+   *
+   */
+  private[sbtdatabricks] def createCluster(
+      name: String,
+      numWorkerContainers: Integer,
+      useSpot: Boolean,
+      sparkVersion: String,
+      zoneId: String): ClusterId = {
```

Comment:
missing doc. In addition, I think it could be an Option[String]. I don't remember it being a required field